companion/src/helpers.cpp

```diff
@@ -681,7 +681,7 @@ void populateSourceCB(QComboBox *b, const RawSource & source, const GeneralSetti
     }
     else {
       for (int i=0; i<(flags & POPULATE_TELEMETRYEXT ? TELEMETRY_SOURCES_STATUS_COUNT : TELEMETRY_SOURCES_COUNT); i++) {
-        if (i==TELEMETRY_SOURCE_RSSI_TX && IS_TARANIS(board))
+        if (i==TELEMETRY_SOURCE_RSSI_TX && IS_TARANIS(board))  // TODO this condition is never true since else clause is never taken for ARM boards!!!
```

Comment:
This is just something I found while working nearby. I think the whole condition it is obsolete and could be removed.